command/command.go

```diff
@@ -23,9 +23,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+func new() {
```

Comment:
Add a usage message to 'registerCompile' to help users understand how to use the command.